### DISCUSSION TITLE
Example of figure with caption, label, and multiple options

### DISCUSSION THREAD

**Discussion by RohanAlexander** [quarto-dev/quarto-cli#574] [Q&A]
Is there an example of a figure with a caption, label, and multiple options in the documentation?

I couldn't see one, but got the following working (a figure that has multiple modifications, and also has a cross-referenced label and caption): 
`![Elephant](elephant.png){#fig-elephant width=50% fig-align="right" fig-alt="A drawing of an elephant."}`

(There's probably a better way to do this; I've got no idea what I'm doing and just trying to hack things together to get it to work.)

**Comment by jjallaire** [comment]
That is in fact the way to do this (lots of attributes on images is I agree pretty syntactically awkward).
